File: scheduler/src/main/java/com/stats/aggregator/config/HadoopConfig.java
package com.stats.aggregator.config;

import org.springframework.context.annotation.Configuration;
import org.springframework.data.hadoop.config.annotation.EnableHadoop;
import org.springframework.data.hadoop.config.annotation.SpringHadoopConfigurerAdapter;
import org.springframework.data.hadoop.config.annotation.builders.HadoopConfigConfigurer;

@Configuration
@EnableHadoop
public class HadoopConfig extends SpringHadoopConfigurerAdapter {

    @Override
    public void configure(HadoopConfigConfigurer config) throws Exception {
        config.fileSystemUri("hdfs://localhost:8021");
    }

}

File: scheduler/src/main/scala/com/stats/aggregator/scheduler/config/HadoopConfig.scala
package com.stats.aggregator.scheduler.config

import org.springframework.context.annotation.Configuration
import org.springframework.data.hadoop.config.annotation.EnableHadoop
import org.springframework.data.hadoop.config.annotation.SpringHadoopConfigurerAdapter
import org.springframework.data.hadoop.config.annotation.builders.HadoopConfigConfigurer

@Configuration
@EnableHadoop class HadoopConfig extends SpringHadoopConfigurerAdapter {

  @throws[Exception]
  override def configure(config: HadoopConfigConfigurer): Unit = {
    config.fileSystemUri("hdfs://localhost:8021")
  }
}